sdk/cognitiveservices/arm-cognitiveservices/rollup.config.js

```diff
@@ -1,122 +0,0 @@
-/*
```

Comment:
forget the package.json here ?

Reply:
deya's commit missed the deletion of this file